setup.py

```diff
@@ -16,6 +16,8 @@
     include_package_data=True,
     package_data={'': ['README.rst']},
     zip_safe=False,
+    install_requires=["django>=2.0.0"],
+    tests_require=["coverage"],
```

Comment:
I can remove these or leave them, but it seemed strange that they don't seem to be declared.